tests/Helper/Input/CheckboxTest.php

```diff
@@ -128,4 +128,25 @@ public function testStrict()
         $expect = '<input type="checkbox" value="1" checked />' . PHP_EOL;
         $this->assertSame($expect, $actual);
     }
+
+    public function testMultiCheckbox() {
+        $checkbox = $this->helper;
+        $actual = $checkbox(array(
+            'name' => 'foo',
+            'value' => 'yes',
+            'options' => array(
+                'yes' => 'Yes',
+                'no' => 'No',
+                'maybe' => 'Maybe'
+            ),
+            'attribs' => array(
+                'value' => 'yes',
+                'label' => 'Is ignored',
```

Comment:
That's right, because we don't have only one label but one label per checkbox (I think Helper\Input\Radio works in the same way, I have just made it explicit in the test)